_extras/guide.md

```diff
@@ -1,8 +1,40 @@
 ---
 layout: page
-title: "Instructors' Guide"
+title: "Instructor Notes"
 permalink: /guide/
```

Comment:
I considered changing the permalink, but after discussions with the mentoring subcommittee we decided the easiest course of action would be to leave the permalink/file names as guide.